.rubocop_todo.yml

```diff
@@ -7,12 +7,12 @@
 
 # Offense count: 8
 Metrics/AbcSize:
-  Max: 51
+  Max: 53
```

Comment:
I would like to disable this offense in Rubocop, and be able to run `--auto-gen-config` for everything else. However when you try to do the latter it ignores what's in `.rubocop.yml`, which defeats the whole purpose.

Feel free to switch it to run after, btw.